build.sbt

```diff
@@ -11,8 +11,8 @@ lazy val commonSettings = Seq(
 sonatypeProfileName := "com.itv"
 
 lazy val noPublishSettings = Seq(
-  publish := (),
-  publishLocal := (),
+  publish := Unit,
+  publishLocal := Unit,
```

Comment:
Surely not?

Reply:
The compiler told me to do it!

Reply:
Well, it said that () was wrong, at least.